js/settingsview.js

```diff
@@ -36,7 +36,7 @@
 		+ '     <span><small>' + t('twofactor_u2f', 'You can add as many devices as you like. It is recommended to give each device a distinct name.') + '</small></span>'
 		+ '	{{else}}'
 		+ '     <span class="icon-loading-small u2f-loading"></span>'
-		+ '	<span>' + t('twofactor_u2f', 'Adding a new device …') + '</span>'
+		+ '	<span>' + t('twofactor_u2f', 'Adding a new device…') + '</span>'
```

Comment:
It's quite simple. If there is an incomplete word like https://, the ellipsis comes directly after. If there is no incomplete word, there should be a space. Hence please add the space here again. :)